### ISSUE TITLE
Broadening options for FieldBook export

### ISSUE THREAD

**Issue by ch728** [solgenomics/sgn#3452]
We store useful meta data for accessions as stock properties. We would like to include some of these properties in our FieldBook export. Is there a way these could be added to the FieldBook export?

**Comment by lukasmueller** [comment]
I think Fieldbook supports importing additional metadata. Currently, our export does not allow to add attributes as columns, but it may not be difficult to add this.

**Comment by ch728** [comment]
Yeah I think FieldBook supports importing whatever column you want. The issue is we store grain type and purdy pedigree strings as additional info for accessions. We would like to use these in the export from breedbase/BrAPI connection. The current Pedigree export option only exports the two most recent parents, but the field people want the full purdy string and additional metadata.
